common/types/types.ts

```diff
@@ -37,4 +37,5 @@ export enum TopicGridSizes {
 
 export type UserData = {
   currentLanguage: Language;
+  favouriteLanguages: Language[];
```

Comment:
We should use either the American or the British spelling of favorite/favourite - not both.